_posts/en/newsletters/2024-11-22-newsletter.md

```diff
@@ -0,0 +1,213 @@
+specification to allow pluggable channel factories, links to a report
+and a new block explorer examining transactions on the default signet
+that use proposed soft forks, describes an update to the LNHANCE
+multi-part soft fork proposal, and discusses a paper about covenants
+based on grinding rather than consensus changes.  Also included are our
+regular sections summarizing recent changes to services, client software,
+and popular Bitcoin infrastructure software.
+
+## News
+
+- **Pluggable channel factories:** ZmnSCPxj [posted][zmnscpxj plug] to
+  Delving Bitcoin a proposal to make a small set of changes to the
+  [BOLT][bolts repo] specification to allow existing LN software to
+  manage [LN-Penalty][topic ln-penalty] payment channels within a
+  [channel factory][topic channel factories] using a software plugin.
+  The specification changes would allow the factory manager (e.g. a
+  Lighting service provider, LSP) to send messages to an LN node that
+  would be passed through to a local factory plugin.  Many factory
+  operations would be similar to [splicing][topic splicing] operations,
+  allowing the plugin to reuse a significant amount of code.  LN-Penalty
+  channel operations within a factory would be similar to [zero-conf
+  channels][topic zero-conf channels], so they could also reuse existing
+  code.
+
+  ZmnSCPxj's design is focused on SuperScalar-style factories (see
+  [Newsletter #327][news327 superscalar]) but would probably be
+  compatible with other factory styles (and possibly other multiparty
+  contract protocols).  Rene Pickhardt [replied][pickhardt plug] to ask
+  about additional specification changes that could allow channels
+  within factories to be [announced][topic channel announcements] but
+  ZmnSCPxj [said][zmnscpxj plug2] he deliberately didn't consider those
+  in his design in order to allow the specification change to be
+  adopted as fast as possible.
+
+- **Signet activity report:** Anthony Towns [posted][towns signet] to
+  Delving Bitcoin a summary of activity on the default [signet][topic
+  signet] related to proposed soft forks available through [Bitcoin
+  Inquisition][bitcoin inquisition repo].  The post looks at
+  [SIGHASH_ANYPREVOUT][topic sighash_anyprevout] usage, including tests
+  of [LN-Symmetry][topic eltoo] and emulation of
+  [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify].  It then looks
+  at `OP_CHECKTEMPLATEVERIFY` usage directly, including what are likely
+  several different [vault][topic vaults] constructions and a few data
+  carrier transactions.  Finally, the post looks at
+  [OP_CAT][topic op_cat] usage, including for a proof-of-work faucet (see
+  [Newsletter #306][news306 powfaucet]), a possible vault or other
+  [covenant][topic covenants], and verification of a [STARK][]
+  zero-knowledge proof.
+
+  Vojtěch Strnad [replied][strnad i.o] that he was inspired by Towns's
+  post to create a [block explorer][topic block explorers] that lists
```

Comment:
Is it really a "block explorer" when it doesn't actually allow exploring blocks and only shows a subset of transactions? I'd slightly prefer just "explorer" or even "website":
```suggestion
  post to create a website that lists
```

Reply:
Updated.